app/code/Magento/Search/view/frontend/web/js/form-mini.js

```diff
@@ -133,6 +133,9 @@ define([
 
             if (this.isExpandable) {
                 this.element.attr('aria-expanded', isActive);
+                let searchValue = this.element.val();
```

Comment:
Hi @sanjaychouhan-webkul. 
Currently, we support only ES5 syntax, so can you please change `let` to `var` and according to our [technical guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html) variables declaration should be on the top of the function.

Reply:
@VladimirZaets I have applied the changes